### DISCUSSION TITLE
Discord official server?

### DISCUSSION THREAD

**Discussion by wolf-yuan-6115** [eritislami/evobot#518] [General]
Can we have a evobot support server?

**Comment by eritislami** [comment]
Why ? you can just use the official discord.js server and ask programming questions there, I can't be expected to provide support for open source project 24/7, Im working a full time job and doing other things, since I don't have much time for it I would rather not create a support server and have people expecting responses.

**Reply by wolf-yuan-6115** [reply]
Hey, but Discord server can support some ~~idiot's question~~

**Comment by Apoo711** [comment]
so many contributors made it though. you could ask them. even if you all have full-time jobs, im sure at least one person will be able to answer. no?
